src/app/components/MovieButton/MovieButton.tsx

```diff
@@ -2,47 +2,52 @@ import React from 'react';
 import BookmarkIcon from '../assets/BookmarkIcon';
 import HomeIcon from '../assets/HomeIcon';
 import SearchIcon from '../assets/SearchIcon';
-import StarIcon from '../assets/StarIcon';
 
-type MovieButtonProps = {
-  iconType:
-    | 'searchActive'
-    | 'searchInactive'
-    | 'homeActive'
-    | 'homeInactive'
-    | 'starActive'
-    | 'starHalfActive'
-    | 'starInactive'
-    | 'bookmarkActive'
-    | 'bookmarkInactive';
+export type MovieButtonProps = {
+  iconType: 'search' | 'home' | 'bookmark';
+  onHandleButtonClick: () => void;
+  toggleActiveState: () => void;
```

Comment:
We won't need this, since this is a controlled component. We can change the active with `onButtonClick`